doc/doc-site/docs/off-chain/python-sdk.md

```diff
@@ -1,3 +1,97 @@
 # Python SDK
 
-More docs coming soon!
+The code for the Python SDK lives in `/econia/src/python/sdk/econia_sdk` ([link](https://github.com/econia-labs/econia/tree/main/src/python/sdk/econia_sdk)). There are 2 primary packages (`econia_sdk.entry` and `econia_sdk.view`) complemented by two secondary imports
+
+# Primary Packages
```

Comment:
There should be more whitespace here, and the formatter should catch it

Reply:
I ran the formatter but nothing changed. What am I supposed to be doing?